docs/source/func.batch_norm.rst

```diff
@@ -0,0 +1,58 @@
+Patching Batch Norm
+===================
+
+What's happening?
+-----------------
+Batch Norm requires in-place updates to running_mean and running_var of the same size as the input.
+Functorch does not support inplace update to a regular tensor that takes in a batched tensor (i.e.
```

Comment:
nit: vmap? And link section from the UX limitations doc if possible (there is some sphinx syntax to link it)

Reply:
I think the UX limitations hasn't been pulled over yet (probably my bad for not looking at a PR yet. Filed #91509 so I don't forget to do this in a follow up)